src/coord_seq.rs

```diff
@@ -724,23 +722,23 @@ impl CoordSeq {
     /// ```
     /// use geos::{CoordDimensions, CoordSeq};
     ///
-    /// let coords = CoordSeq::new(2, CoordDimensions::OneD)
+    /// let coords = CoordSeq::new(2, CoordDimensions::ThreeD)
```

Comment:
Isn't it two dimensions?

Reply:
The 2 on the left is the length of the coordinate sequence, not its dimension. This line just creates a new coordinate sequence with dimension 3, and confirm on the next line that it has been created properly.